webpack/dev.config.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+const webpack = require('webpack');
+var webpackMerge = require('webpack-merge');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var commonConfig = require('./common.config.js');
+const IsomorphicPlugin = require('webpack-isomorphic-tools/plugin');
+const path = require('path')
+const webpackIsomorphicToolsPlugin = new IsomorphicPlugin(require('./isomorphic.config'));
+
+module.exports = webpackMerge(commonConfig, {
+  devtool: 'cheap-module-eval-source-map',
+  output: {
+    path: path.resolve('./build'),
+    filename: '[name].js',
+    chunkFilename: '[id].chunk.js'
+  },
+
+  plugins: [
+    new ExtractTextPlugin('[name].css'),
+    new webpack.ProvidePlugin({
+      $: 'jquery',
```

Comment:
The react community has mixed feelings about using jQuery and React afaik. You can check this [discussion](https://discuss.reactjs.org/t/jquery-with-react/683/4) but it's entirely up to you, although I think you can have a jquery-free react application. The presence of jQuery just brings in a different paradigm, and kills uniformity. Moving forward, devs would do things with jQuery that could otherwise have been done with the React-Redux flow, or React-Flux flow.